launch-script/src/proposals/create_treasury_pool.rs

```diff
@@ -0,0 +1,75 @@
+// =========================================================================
+// Create collateral pool accounts
+// =========================================================================
+
+use crate::prelude::*;
+
+const TREASURY_POOL_ACCOUNT_COUNT: u32 = 10;
+
+pub fn create_collateral_pool_accounts(wallet: &Wallet, client: &Client, transaction_inserter: &mut ProposalTransactionInserter, cfg: &Configuration) -> Result<(), ScriptError> {
+
+    let realm = Realm::new(
+        cfg.client,
+        &wallet.governance_program_id,
+        REALM_NAME,
+        &wallet.community_pubkey,
+    );
+    let maintenance_governance = realm.governance(&wallet.neon_evm_program_id);
+    let maintenance_governance_pubkey: Pubkey = maintenance_governance.governance_address;
+    
+    let (maintenance_record_pubkey,_): (Pubkey,u8) =
+        get_maintenance_record_address(&wallet.maintenance_program_id, &wallet.neon_evm_program_id);
+    
+    println!("Maintenance DAO Address: {:?}", maintenance_governance_pubkey);
+    println!("Maintenance Record Address: {:?}", maintenance_record_pubkey);
+    
+
+    let minimum_balance_for_rent_exemption = cfg.client.get_minimum_balance_for_rent_exemption(0).unwrap();
+
+    let executor = TransactionExecutor {
+        client,
+        setup: transaction_inserter.setup,
+        verbose: transaction_inserter.verbose,
+    };
+
+    for index in 0u32..TREASURY_POOL_ACCOUNT_COUNT {
+        let seed: String = format!("collateral_seed_{}", index.to_string().as_str());
```

Comment:
Why do you need `index.to_string().as_str()`? It is worked good using simply `index`.

Reply:
Fixed